remoteworkitem/github.go

```diff
@@ -45,6 +45,8 @@ func (g *GithubTracker) fetch(f githubFetcher) chan TrackerItemContent {
 	item := make(chan TrackerItemContent)
 	go func() {
 		opts := &github.SearchOptions{
+			Sort:  "updated",
```

Comment:
Sorting by is fine, but it doesn't really fix half the problem.

You can only fetch a 1000 items at a time with pagination.

Even with sort you'll only get the first 1000 items unless something is changed, over and over again.

You need to track 'what was the updated time of the last item you imported' then use that as part of the query 'list items updated after previous last updated time'.

With that you shouldn't be getting back unchanged items from GitHub either.